tests/pom.xml

```diff
@@ -72,7 +72,14 @@
     <!-- Internal test dependencies -->
     <dependency>
       <groupId>org.alluxio</groupId>
-      <artifactId>alluxio-core-client</artifactId>
+      <artifactId>alluxio-core-client-hdfs</artifactId>
+      <version>${project.version}</version>
+      <type>test-jar</type>
+      <scope>test</scope>
+    </dependency>
+    <dependency>
+      <groupId>org.alluxio</groupId>
+      <artifactId>alluxio-core-client-native</artifactId>
```

Comment:
We depend on the source code here, whereas for the `hdfs` client we only need the test utility methods (though I think we probably can get away with not using them if we clean things up later).